astroquery/atomic/core.py

```diff
@@ -140,6 +140,10 @@ def query_object(self, *, wavelength_range=None, wavelength_type=None, wavelengt
             Angular momentum, Transition probability and Level energies
             respectively.
 
+        cache : bool
+            Defaults to True. If set overrides global caching behavior.
+            See [caching documentation](https://astroquery.readthedocs.io/en/latest/index.html#caching).
```

Comment:
Do use a sphinx reference for this rather than the full URL, here and everywhere else, too.